src/main/java/racingcar/Validator/CarNameValidator.java

```diff
@@ -0,0 +1,46 @@
+package racingcar.Validator;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CarNameValidator {
+    private static final int maxCarCount = 9;
+    private static final int maxCarName = 5;
+    private static final String commaSeparator = ",";
+
+    public static List<String> validateCarNames(String carNamesInput) {
+        if (isNotSplitByComma(carNamesInput)) {
+            throw new IllegalArgumentException("자동차는 쉼표(,)로 구분해주세요.");
+        }
+
+        List<String> carNames = splitByComma(carNamesInput);
+
+        for (String carName : carNames) {
+            if (isInvalidCarName(carName)) {
+                throw new IllegalArgumentException("자동차 이름은 1자 이상, 5자 이하만 가능합니다.");
+            }
+
+            if (carNames.isEmpty() || carNames.size() > maxCarCount) {
```

Comment:
이런식으로 메서드의 순서를 눈에 잘 보이게 나타내면 더 좋을것같아요!
해당 메서드도 위 isInvalidCarName() 처럼 메서드로 꺼내도 좋을 것 같네요~
```suggestion
            if ((carNames.isEmpty()) || (carNames.size() > maxCarCount)) {
```